app/src/main/java/com/example/solar_alarm/AlarmAdapter.java

```diff
@@ -51,6 +55,14 @@ public void onBindViewHolder(@NonNull AlarmViewHolder holder, int position)
 
         textView = holder.TimeTextView;
         textView.setText(alarm.GetAlarmTime().toString());
+        holder.parent_layout.setOnClickListener(new View.OnClickListener() {
```

Comment:
This is applying to all the alarms in the list. It should bring the user to an edit screen because it is an existing alarm. We still have the mocked alarms created on the fly at startup.

Reply:
Let's do this on another PR that focuses on editing alarms.